app/helpers/wally_integration.py

```diff
@@ -54,4 +55,19 @@ def create_wallet(user: User):
   response = requests.post(request_url, headers=headers, json=payload)
   response.raise_for_status()
   data = response.json()
-  return data
\ No newline at end of file
+  return data
+
+def sign_message(user: User, dive_shop: DiveShop):
+  wallet_id = f'user_{str(user.id)}'
```

Comment:
this should be the dive shops wallet not the users wallet. so it should be `f'shop_{str(dive_shop.id)}'`

Reply:
Ah, got it. My bad.